orangecontrib/text/widgets/owtwitter.py

```diff
@@ -296,6 +298,7 @@ def update_api(self, key):
 
     @QtCore.pyqtSlot()
     @require('api', MISSED_KEY, MISSED_KEY_MESSAGE)
+    @require('word_list', MISSED_QUERY, MISSED_QUERY_MESSAGE)
```

Comment:
I cannot execute a query without any input in the `Query word list`. I am getting the error with code 403 and hence I added the check. 

Would this supposed to work, i.e. does tweepy even allow for request without any query and only time restrictions? Does this work for you?